### PR TITLE
fix some en doc

### PR DESCRIPTION
**Pull request by wangchaochaohu**
create_tensor

image

fill_constant
image

fill_constant_batch_size_like

image

strided_slice
image

image

equal
image

PipeReader.get_line
image
image

### CODE REVIEW

File: python/paddle/fluid/layers/tensor.py [hunk 1]
```diff
@@ -381,20 +382,28 @@ def fill_constant(shape, dtype, value, force_cpu=False, out=None):
     The attribute `stop_gradient` of the created tensor is set to True.
 
     Args:
-        shape(tuple|list|None): Shape of the output tensor.
-        dtype(np.dtype|core.VarDesc.VarType|str): Data type of the output tensor.
-        value(float): The constant value used to initialize the output tensor.
-        out(Variable): The output tensor.
-        force_cpu(True|False): data should be on CPU if set true.
+        shape(tuple|list): Shape of the LoDTensor or SelectedRows to be created.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py]: Shape of the LoDTensor or SelectedRows to be created.
与之前的描述不符合
This function creates ** a tensor ** with specified `shape` and `dtype`

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -1165,20 +1165,27 @@ def equal(x, y, cond=None):
...
 
     Args:
-        x(Variable): First operand of *equal*
-        y(Variable): Second operand of *equal*
-        cond(Variable|None): Optional output variable to store the result of *equal*
+        x(Variable): Tensor, data type is float32, float64, int32, int64.
+        y(Variable): Tensor, data type is float32, float64, int32, int64.
+        cond(Variable, optional): Optional output which can be any created 
+            Variable that meets the requirements to store the result of *equal*.
+            if cond is None, a new Varibale will be create to store the result.
```
Comment (cyj1986) [inline python/paddle/fluid/layers/control_flow.py]: a new Varibale will be created to store the result

File: python/paddle/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -1165,20 +1165,27 @@ def equal(x, y, cond=None):
...
 
     Returns:
-        Variable: The tensor variable storing the output of *equal*.
+        Variable: output Tensor that the shape is the same as the input Tensor,
```
Comment (cyj1986) [inline python/paddle/fluid/layers/control_flow.py]: output Tensor,  its shape  is the same ad input's

File: python/paddle/fluid/layers/tensor.py [hunk 4]
```diff
@@ -381,20 +382,28 @@ def fill_constant(shape, dtype, value, force_cpu=False, out=None):
...
+        dtype(np.dtype|core.VarDesc.VarType|str): Data type of the output tensor which can
+            be float16, float32, float64, int32, int64.
+        value(float): The constant value used to initialize the LoDTensor or SelectedRows to be created.
+        force_cpu(True): data should be on CPU if set true, defalut value is False.
```
Comment (cyj1986) [inline python/paddle/fluid/layers/tensor.py]: if set true ->if it's true

File: python/paddle/fluid/layers/tensor.py [hunk 5]
```diff
@@ -381,20 +382,28 @@ def fill_constant(shape, dtype, value, force_cpu=False, out=None):
...
+            Variable that meets the requirements to store the result of operation.
+            if out is None, a new Varibale will be create to store the result.
 
     Returns:
-        Variable: The tensor variable storing the output.
+        Variable: LoDTensor or SelectedRows which is created according the shape and dtype.
+
+    Raise:
+        TypeError: The dtype must be one of bool, float16, float32, float64, int32 and int64
+        and the data type of out Tensor must be same as the dtype.
```
Comment (cyj1986) [inline python/paddle/fluid/layers/tensor.py]: be same as ->be the same as

File: python/paddle/fluid/layers/tensor.py [hunk 6]
```diff
@@ -436,34 +445,36 @@ def fill_constant_batch_size_like(input,
...
+        value(float|int): The constant value used to initialize the Tensor to be created. 
+        input_dim_idx(int): When the value is 0 and the input is LoDTensor, the output_dim_idx
+            dimension of the created Tensor is set to the batch_size value of input.
+            The default value is 0.
+        output_dim_idx(int): Used to specify which dimension of Tensor is created to be set
+            the value of batch_size of input Tensor. The default value is 0.
 
     Returns:
-        ${out_comment}.
+        Variable: the created Tensor which data type si dtype.
```
Comment (cyj1986) [inline python/paddle/fluid/layers/tensor.py]: the created tensor whose data type is dtype

File: python/paddle/fluid/layers/tensor.py (hunk 4 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py]: Done

File: python/paddle/fluid/layers/tensor.py (hunk 5 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py]: Done

File: python/paddle/fluid/layers/tensor.py (hunk 6 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py]: Done

File: python/paddle/fluid/layers/control_flow.py (hunk 2 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/control_flow.py]: Done

File: python/paddle/fluid/layers/control_flow.py (hunk 3 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/control_flow.py]: Done

File: python/paddle/fluid/layers/tensor.py [hunk 7]
```diff
@@ -35,17 +35,18 @@
...
     """
-    Create an variable, which will hold a LoDTensor with data type dtype.
+    Create a variable, which will hold a LoDTensor with data type dtype.
 
     Args:
-        dtype(string): 'float32'|'int32'|..., the data type of the
-            created tensor.
-        name(string): The name of the created tensor, if not set,
-            the name will be a random unique one.
+        dtype(string|numpy.dtype): the data type of Tensor to be created, the
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py:41]: the data type of Tensor to be created
与前面描述不符
Create a variable, which will hold a **LoDTensor ** with data type dtype.
其他地方也检查一下

File: python/paddle/fluid/layers/tensor.py [hunk 8]
```diff
@@ -35,17 +35,18 @@
...
+            data type is bool, float16, float32, float64, int8, int16, int32 and int64.
+        name(string, optional): The default value is None.  Normally there is no need for 
+            user to set this property.  For more information, please refer to :ref:`api_guide_Name`
         persistable(bool): Set the persistable flag of the create tensor.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py:45]: 同上

File: python/paddle/fluid/layers/tensor.py [hunk 9]
```diff
@@ -35,17 +35,18 @@
...
+            default value is False.
 
     Returns:
-        Variable: The tensor variable storing the created tensor.
+        Variable: The tensor to be created according to dtype.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py:49]: 同上

File: python/paddle/fluid/layers/tensor.py [hunk 10]
```diff
@@ -373,28 +374,34 @@ def assign(input, output=None):
...
+        TypeError: The dtype must be one of bool, float16, float32, float64, int32 and int64
+        and the data type of out Tensor must be the same as the dtype. 
 
     Examples:
         .. code-block:: python
 
           import paddle.fluid as fluid
-          data = fluid.layers.fill_constant(shape=[1], value=0, dtype='int64')
+          data1 = fluid.layers.fill_constant(shape=[1], value=0, dtype='int64') #data1=[0]
+          data2 = fluid.layers.fill_constant(shape=[1], value=5, dtype='int64', out=data1) #data1=[5] data2=[5]
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py]: 需要补充创建SelectedRows 的示例

File: python/paddle/fluid/layers/tensor.py [hunk 11]
```diff
@@ -436,34 +443,36 @@ def fill_constant_batch_size_like(input,
...
                                   output_dim_idx=0):
     """
-    ${comment}
-
-    It also sets *stop_gradient* to True.
+    This OP creates a Tesnor accroding the shape and dtype, and initializes the
+    Tensor with the constants provided in ``value``. When the input is LoDTensor
+    and the input_dim_idx is 0, the output_dim_idx dimension is set to the value
+    of the batch_size input by the input, the Stop_gradient attribute of the created
+    Tensor is False by default.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py:451]: 同上，说清楚Tensor和LoDTensor

Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py:451]: 此处是Tensor

File: python/paddle/fluid/layers/tensor.py (hunk 10 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py]: 已加示例

File: python/paddle/fluid/layers/tensor.py (hunk 9 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py:49]: 此处是Tensor

File: python/paddle/fluid/layers/tensor.py (hunk 1 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py]: 已改

File: python/paddle/fluid/layers/tensor.py (hunk 8 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py:45]: Done

File: python/paddle/fluid/layers/tensor.py (hunk 7 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py:41]: 已改

**Review by cyj1986 — DISMISSED** [review]
LGTM

**Review by heavengate — DISMISSED** [review]

**Review by hong19860320 — DISMISSED** [review]
LGTM

**Review by cyj1986 — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**